StudentDetails.java

```diff
@@ -0,0 +1,155 @@
+import java.util.*;
+import java.util.Scanner;
+
+class StudentData {
+    private int sno;
+    private String sfname;
+    private String slname;
```

Comment:
The same applies here - avoid abbreviations. Just use __lastName__ instead of __slname__.